src/org/folio/rest_v2/Edge.groovy

```diff
@@ -83,6 +83,10 @@ ${institutional}
 
             if (edgePermissions) {
                 edgePermissionsList.addAll(edgePermissions)
+                if (edgePermissionsList.contains('user-tenants.collection.get')) {
```

Comment:
Nope) Changes should be made in https://github.com/folio-org/pipelines-shared-library/blob/RANCHER-741-Jenkins-Enhancements/resources/edge/config.yaml#L6